### ISSUE TITLE
As HeadsPluginAPI developer I would like to be able to have the bytecode match the source, so that I can debug the code properly

### ISSUE THREAD

**Issue by CC007** [Alan-Gomes/mcspring-boot#28]
I saw that you are using lombok in your source code. This causes issues while debugging, because the source code doesn't match up with the bytecode in the .class file. 

One way to fix this is to run the delombok with the lombok-maven-plugin, so that the sources in the artefact sources don't contain lombok.

**Comment by Alan-Gomes** [comment]
Hello, delombok is already included in the project:

https://github.com/Alan-Gomes/mcspring-boot/blob/master/pom.xml#L198-L215

Maybe you're not downloading the dependency sources.

**Comment by CC007** [comment]
This is true, but while you do use this lombok output for your maven-javadoc-plugin, you do not use it for your maven-source-plugin

**Comment by CC007** [comment]
Here is a good explanation on how to do this:

https://stackoverflow.com/questions/52362413/create-and-install-de-lomboked-source-jar-in-maven

The first answer lays the foundation and the second answer makes it more generally usable
